src/hotspot/share/jeandle/jeandleCompiledCode.cpp

```diff
@@ -205,6 +205,39 @@ void JeandleCompiledCode::install_obj(std::unique_ptr<ObjectBuffer> obj) {
   }
 }
 
+void JeandleCompiledCode::estimate_buffer_size(std::unique_ptr<llvm::jitlink::LinkGraph> &link_graph, int& const_size, int& stubs_size) {
+  for (auto *block : link_graph->blocks()) {
+    // Only resolve relocations for instructions in the compiled method.
+    if (block->getSection().getName().compare(".text") != 0) {
+      continue;
+    }
+    for (auto& edge : block->edges()) {
+      auto& target = edge.getTarget();
+      if (target.isDefined() && JeandleAssembler::is_const_reloc_kind(edge.getKind())) {
+        // Const relocations.
+        const_size += 8;
+      }
+    }
+  }
+  // reserve a space for alignment
+  const_size += 8;
+  // Step 2: Resolve stackmaps.
```

Comment:
```suggestion
  // Step 2: Resolve stackmaps.
```
Complete the comments of `Step 1`, or remove the `Step2:`